tests/kolainst/destructive/layering-fedorainfra

```diff
@@ -8,13 +8,12 @@ set -euo pipefail
 cd $(mktemp -d)
 
 # bodhi update for rpm-ostree (Fedora 33)
-rpm-ostree override replace https://bodhi.fedoraproject.org/updates/FEDORA-2020-6e743def1d
+rpm-ostree override replace https://bodhi.fedoraproject.org/updates/FEDORA-2021-e55da2fc78
 rpm-ostree status > status.txt
```

Comment:
Do we still need this then?

Reply:
Ah no we don't...will remove it in a later PR to avoid respinning CI here.